platforms/core-configuration/configuration-cache/src/main/kotlin/org/gradle/configurationcache/EncryptionService.kt

```diff
@@ -185,19 +178,37 @@ class DefaultEncryptionService(
         encryptionAlgorithm.newSession(secretKey)
 
     private
-    fun secretKeySource() = KeyStoreKeySource(
```

Comment:
There are three possible sources of keys: keystore, env var, and a no-op source (used in tests)